### PR TITLE
Add request count to watchlist tests

### PR DESCRIPTION
**Pull request by marseel**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Let's add request count to watchlist tests as well. 
Additionally switch local testing of perfdash from using githubConfig to local config.

### CODE REVIEW

File: perfdash/config.go [hunk 1]
```diff
@@ -658,6 +658,11 @@ var (
 				OutputFilePrefix: "APIResponsivenessPrometheus_simple",
 				Parser:           parsePerfData,
 			}},
+			"LoadRequestCount_PrometheusSimple": []TestDescription{{
```
Comment (p0lyn0mial) [inline perfdash/config.go:661]: that's awesome, thanks!

File: perfdash/Makefile [hunk 2]
```diff
@@ -19,7 +19,7 @@ run: perfdash
 		--address=0.0.0.0:8080 \
 		--builds=20 \
 		--force-builds \
-		--githubConfigDir=https://api.github.com/repos/kubernetes/perf-tests/contents/perfdash/test
+		--configPath=test/jobs.yaml
```
Comment (p0lyn0mial) [inline perfdash/Makefile:22]: ok, this changes the cfg for a local binary.

Comment (marseel) [reply perfdash/Makefile:22]: yes, just for testing easier :)

**Pull request merged**